plugins/router/f5/f5.go

```diff
@@ -712,6 +825,11 @@ func (f5 *f5LTM) Initialize() error {
 		}
 	}
 
+	err = f5.ensurePartitionPathExists(f5.partitionPath)
```

Comment:
We need to call `f5.ensurePartitionPathExists(f5.partitionPath)` before calling `ensureVserverHasIRule` because the latter refers to `f5.partitionPath` when referring to the iRule.

In fact, I suspect we need to specify `f5.partitionPath` explicitly in a lot of places (including `ensurePolicyExists`, `